tests/integration/test_suite_hierarchy.py

```diff
@@ -0,0 +1,271 @@
+        {'name': 'test_in_class_parameterized.py', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith('examples')},
+        {'name': 'Tests', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(
+             'test_in_class_parameterized.py')},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith('Tests')}
+    ],
+    [
+        {'name': 'examples/test_in_class_parameterized.py::Tests',
+         'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(
+             'examples/test_in_class_parameterized.py::Tests')}
+    ],
+    [
+        {'name': 'examples', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': 'test_in_class_parameterized.py::Tests', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith('examples')},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(
+             'test_in_class_parameterized.py::Tests')}
+    ],
+    [
+        {'name': 'examples/test_in_class_parameterized.py::Tests',
+         'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': 'test_in_class_parameterized', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(
+             'examples/test_in_class_parameterized.py::Tests')},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(
+             'test_in_class_parameterized')}
+    ],
+    [
+        {'name': 'Tests',
+         'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith('Tests')}
+    ],
+    [
+        {'name': 'test_in_class_parameterized.py', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': 'Tests', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(
+             'test_in_class_parameterized.py')},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith('Tests')}
+    ],
+    [
+        {'name': 'examples\\test_in_class_parameterized.py',
+         'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': 'Tests', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(
+             'examples\\test_in_class_parameterized.py')},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith('Tests')}
+    ]
+]
+
+TEST_PARAMETERS = [(v, TEST_EXPECTED_ITEMS[i]) for i, v in
```

Comment:
Can you please rename single character variables to something that may give an idea of their content?